robottelo/virtwho_utils.py

```diff
@@ -407,22 +410,23 @@ def virtwho_package_locked():
     assert "Packages are locked" in result[1]
 
 
-def create_http_proxy(name=None, url=None, type='https'):
+def create_http_proxy(name=None, url=None, http_type='https', org_name=DEFAULT_ORG):
     """
     Creat a new http-proxy with attributes.
     :param name: Name of the proxy
     :param url: URL of the proxy including schema (https://proxy.example.com:8080)
-    :param type: https or http
+    :param http_type: https or http
+    :param org_name: name of the organization
     :return:
     """
-    default_org = entities.Organization().search(query={'search': f'name="{DEFAULT_ORG}"'})[0]
+    org = entities.Organization().search(query={'search': f'name="{org_name}"'})[0]
     http_proxy_name = name or gen_string('alpha', 15)
     http_proxy_url = url or '{}:{}'.format(
```

Comment:
Can you try use f string here? I notice there are quite a few ".format" in the file, a job for another PR to fix them.

Reply:
Good catch! Copied that from another file before, updated.